plugins/microshift-prow-jobs/commands/get-scenario-info.md

```diff
@@ -0,0 +1,462 @@
+4. **Parsing test results** to extract pass/fail counts, test case names, and failure details
+5. **Extracting scenario metadata** from the scenario name (RHEL version, release type, test category)
+6. **Compiling artifact links** for all logs and diagnostic files
+7. **Generating structured JSON output** containing all collected information
+
+If no scenario name is provided, the command lists all available scenarios by fetching the scenario-info directory listing from GCS.
+
+The command uses the `extract_microshift_version.py` helper script to determine the exact MicroShift version tested in the scenario.
+
+## Arguments
+- `$1` (job-url): URL to the Prow CI job - **Required**
+  - Formats accepted:
+    - Full Prow dashboard URL: `https://prow.ci.openshift.org/view/gs/test-platform-results/logs/<job-name>/<job-id>`
+    - GCS web URL: `https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/<job-name>/<job-id>`
+- `$2` (scenario-name): Name of the scenario to analyze - **Required**
+  - Examples: `el96-lrel@standard1`, `el96-lrel@lvm`, `el96-lrel@dual-stack`
+  - If not provided, the command will list all available scenarios
+
+## Return Value
+- **Format**: JSON object
+- **Location**: Output directly to the conversation
+- **Content**: Comprehensive scenario information including test results, configuration, and artifacts
+
+## Implementation Steps
+
+### Step 1: Parse and Validate Input
+
+**Goal**: Extract job information and scenario name from the arguments.
+
+**Actions**:
+1. Parse the job URL to extract:
+   - Job name
+   - Job ID
+   - Version (e.g., "4.20")
+   - Job type (bootc/rpm-ostree, x86_64/aarch64)
+2. Validate scenario name format (should match pattern: `el[0-9]+-[a-z0-9]+@.+`)
+3. If no scenario name provided, set `list_scenarios = true` flag
+
+**Example**:
+```javascript
+// Input
+job_url = "https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-microshift-release-4.20-periodics-e2e-aws-tests-bootc-release-periodic/1979744605507162112"
+scenario_name = "el96-lrel@standard1"
+
+// Parsed
+job_id = "1979744605507162112"
+job_name = "periodic-ci-openshift-microshift-release-4.20-periodics-e2e-aws-tests-bootc-release-periodic"
+version = "4.20"
+job_type = "e2e-aws-tests-bootc-release-periodic"
+arch = "x86_64"
+image_type = "bootc"
+```
+
+### Step 2: Construct Artifact URLs
+
+**Goal**: Build URLs to the scenario's artifacts.
+
+**Actions**:
+1. Construct base artifact URL:
+   ```
+   https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/<job-name>/<job-id>/artifacts/<job-type>/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/<scenario-name>/
+   ```
+2. Construct specific artifact URLs:
+   - JUnit XML: `<base-url>/junit.xml`
+   - Boot log: `<base-url>/boot_and_run.log`
+   - Debug log: `<base-url>/rf-debug.log`
+   - Phase logs: `<base-url>/phase_*/*.log`
```

Comment:
Most of the prow job tools have settled on using gcs utils to browse the buckets instead, its a little faster I think than trying to navigate the gcsweb interface